internal/exec/stages/files/filesystemEntries.go

```diff
@@ -140,20 +140,34 @@ func (s *stage) createResultFile() error {
 	}
 
 	s.Logger.PushPrefix("createResultFile")
+	var prevReport interface{}
+	prevPath, _ := s.JoinPath(distro.ResultFilePath())
+
+	prevData, err := ioutil.ReadFile(prevPath)
+	if err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("reading previous report: %w", err)
+	}
+	if err == nil {
+		err = json.Unmarshal(prevData, &prevReport)
```

Comment:
Actually, let's log a warning in this case too, similar to the one in https://github.com/coreos/fedora-coreos-config/pull/1148.

Reply:
Shouldn't we only do this only if we're not on a live system? Otherwise, this will be forever stacking `previousReport`s on each live boot.

Reply:
> Shouldn't we do this only if we're not on a live system? Otherwise, this will be forever stacking `previousReport`s on each live boot.

Yeah, that's a good point!

Reply:
Each of the stacked Ignition runs affect the live system, so my thinking was that recording the run history was useful.  But I'm okay dropping it if we want.

Reply:
I'm OK adding onto it I guess, though continually nesting when you know it could keep going forever by design feels weird. Would be more natural if it were a list instead, i.e. `previousReports[]`.